### ISSUE TITLE
weird behavior on caching code in rails console

### ISSUE THREAD

**Issue by jnylund** [deivid-rodriguez/byebug#120]
I have a ruby file, when I put "debugger" in it, it stops fine, but when I changed the file and moved the debugger statement down a few lines, byebug seems to stop at the old breakpoint. It shows me the new file, but when I step over it, it seems that it is showing some type of cached version  of it?

Does byebug do any caching I need to clean up?

thanks
Joel

**Comment by deivid-rodriguez** [comment]
Yes, this is an open issue I need to handle. I'll leave this opened.

**Comment by jnylund** [comment]
Thanks, any work around?

**Comment by deivid-rodriguez** [comment]
This used rely on an external gem (debugger-linecache) in previous versions (maybe 3.5.1 too), but it was not really working so I removed the gem and tried to implement it from scratch but I haven't managed to do it yet. In previous versions there was this `reload` command. Try it, maybe it works.

**Comment by jnylund** [comment]
Tried, didnt work (or wasnt found), I will wait, thanks!
